tests/propertylookup/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -24,4 +31,15 @@ $b = $this->{'foo' . $bar};
 $b = $this->{self::STUFF};
 $object = $this->{'bar' . $foo}->buzz();
 
+$this->loooooooooooongloooooooooooongloooooooooooonglookup =
+    "teeeeeeeeeeeeeeeeeeeeest";
+$this->loooooooooooong->loooooooooooong->loooooooooooong->lookup =
+    "teeeeeeeeeeest";
+$this->loooooooooooong->loooooooooooong->loooooooooooong->lookup = $other->
```

Comment:
How about this? https://prettier.io/playground/#N4Igxg9gdgLgprEAuEASGALAlgZwHQA2ExJpxUA5oWTdFUbaZdRANYCuADgAQC83qCJjgAnFozp4cMdgDNZAbhAAaEBE4ws0HMlABDESIgB3AAoGEOlHoBuELABMVIPdOSy9BHHFUAjEXpgrHAwAMqcgViUyDAi7D4gDhBg7p7eqlHeIjCmARQAtnqpXgkAVjgAHgBCAUEhoXr5cAAyUXDF6WrsMJzdAEwdCREiWcggvnq+AJ5EUM6cIlEwAOqOmMgAHAAMqgsQ3ssBnGMLcFk27aoicACO7FjXuXoFRUgeJare+VgxcQk4UQoBDgAEV2EJ2m80gkYJNVg51kg+qpYnosARAQBhCD5QpjKDQS4gdjeAAqkys728AF9qUA

Reply:
:+1: Added it as in the JS printer.